get_started/what_is_citus.rst

```diff
@@ -66,7 +42,11 @@ For more customers and statistics, see our `customer stories <https://www.citusd
 When to Use Citus
 =================
 
-Citus serves many use cases. Two common ones are scaling multi-tenant (B2B) databases and real-time analytics. In addition to the information below, there are examples of Citus `use-cases <https://www.citusdata.com/use-cases>`_ and `customer case studies <https://www.citusdata.com/solutions/case-studies>`_ on our main web site.
+Citus serves many use cases. Two common ones are scaling multi-tenant (B2B) databases and real-time analytics.
```

Comment:
Could we add some verbiage here like:
"Citus provides a lot of benefits for these use-cases, and below are some of the reasons why Citus is the best choice for these use-cases" (An amended version of that).

Another option is the move the below 2 paragraphs into a "Why Citus" right before "When to use".

Reply:
I was trying various things moving that content around, and it just felt redundant and long-winded. See how you like it with the paragraphs removed.